lv_conf_template.h

```diff
@@ -176,6 +176,9 @@
 /* Enable VG-Lite assert. */
 #define LV_VG_LITE_USE_ASSERT 0
 
+/* VG-Lite flush commit trigger threshold */
```

Comment:
```suggestion
/* VG-Lite flush commit trigger threshold. VGlLite will try to batch these many draw tasks */
```

- How can we determine the ideal value? Can it be 256 too? 
- How large the command buffer is?

Reply:
> How can we determine the ideal value? Can it be 256 too?

8 is an experience value. I plan to implement this set of logic in the underlying driver in the future. Check whether the GPU is in idle state in vg_lite_flush: if the GPU is in idle, the rendering will be submitted immediately. If the GPU is busy, the rendering task will be queued to wait for the next time. examine.

> How large the command buffer is?

Currently we configure the cmd buffer to be 128K x2. These two buffers are ping-pong buffers. When the GPU is reading one of the buffers, the user can write to the other buffer and swap at the appropriate time.